packages/wonder-blocks-modal/docs.md

```diff
@@ -37,3 +37,70 @@ class ModalWrapper extends React.Component {
     </ModalLauncher>
 </View>
 ```
+
+### Example: Modals with custom styles 
+
+Sometimes you'll want to customize the styling of the modal .e.g., custom width or height.  You can pass in `style` which will customize the styling of the `ModalDialog`.
+To use styling for different screen sizes, wrap your component with `MediaLayout` component.  Please see example code below for details.
+
+```js
+const {StyleSheet} = require("aphrodite");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const {Title, Body} = require("@khanacademy/wonder-blocks-typography");
+const {MediaLayout} = require("@khanacademy/wonder-blocks-layout");
+
+const styles = StyleSheet.create({
+    previewSizer: {
+        height: 512,
+    },
+
+    modalPositioner: {
+        // Checkerboard background
+        backgroundImage: "linear-gradient(45deg, #ccc 25%, transparent 25%), linear-gradient(-45deg, #ccc 25%, transparent 25%), linear-gradient(45deg, transparent 75%, #ccc 75%), linear-gradient(-45deg, transparent 75%, #ccc 75%)",
+        backgroundSize: "20px 20px",
+        backgroundPosition: "0 0, 0 10px, 10px -10px, -10px 0px",
+
+        display: "flex",
+        flexDirection: "row",
+        alignItems: "center",
+        justifyContent: "center",
+
+        position: "absolute",
+        left: 0,
+        right: 0,
+        top: 0,
+        bottom: 0,
+    },
+});
+
+const styleSheets = {
+    all: StyleSheet.create({
```

Comment:
I think this example should use `mdOrLarger` since currently our modals on mobile go full-screen and my guess is that people will probably want to maintain that behavior.